app/src/main/res/values/servers.xml

```diff
@@ -16,8 +16,13 @@
         <item>9299</item>
     </string-array>
 
+    <string-array name="content_server_data_server_values_array">
+        <item>https://wacs.bibletranslationtools.org</item>
+        <item>https://git.door43.org</item>
+    </string-array>
+
     <string-array name="content_server_git_server_api_values_array">
-        <item>https://content.bibletranslationtools.org/api/v1</item>
+        <item>https://wacs.bibletranslationtools.org/api/v1</item>

```

Comment:
Please change this back to `content`.  Although we support `wacs.bibletranslationtools.org` for backwards compatibility, `content` is the URL supported by CloudFlare.

Reply:
Done.